### PR TITLE
Add note on how to run tests

### PR DESCRIPTION
**Pull request by rossf7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,14 @@ go build github.com/giantswarm/awstpr
 However, since this project is just a specification used by the other projects, there only goal of building is to check
 whether the build is successful. This is just a library which needs to be vendored by the project aiming to use it.
 
+### Run the tests
+
+To run the tests.
+
+```
+go test $(glide novendor)
```
Comment (JosephSalisbury) [inline README.md]: With golang 1.9.0, we don't need glide anymore - `./...` doesn't match vendor.

So, this can totally just read `go test ./...`

Comment (rossf7) [reply README.md]: Done. Good catch that's really nice! <3

Comment (JosephSalisbury) [reply README.md]: My favourite feature in 1.9 :D

Comment (fgimenez) [reply README.md]: Nice! :100:

**Pull request merged**